docs/how-to/hsm.md

```diff
@@ -0,0 +1,45 @@
+# Use an HSM as Notary's Encryption Backend
+
+In this guide we show the required configuration to use a Hardware Security Module (HSM) as an encryption backend for Notary.
+
+**Note**: 
+- Once Notary is initialized it must continue using the encryption backend configured at the time of initialization, at the moment there is no way to switch backends.
+- This has been tested with YubiHSM2, while this should work with any HSM that supports the PKCS11 protocol please follow any further instruction from the specific HSM vendor.
+
+## Hardware Security Modules (HSMs)
+
+### Prerequisites:
+- An HSM that supports the PKCS11 protocol
+- AES256 symmetric key created on the HSM with capabilities to encrypt and decrypt using the AES-CBC algorithm
+- Access to the HSM's driver/interface library file (.so or .dylib file) installed with the HSM's SDK
+- The HSM's connector up and running
+
+### Configuration
+If you are using the snap you can modify the config under `/var/snap/notary/common/notary.yaml`
```

Comment:
I think this line belongs in the config file reference, we don't want to add such a line in every how-to guide that relates to managing the config file.

Reply:
moved